frontend/src/components/Summary/ImplementationRow.jsx

```diff
@@ -70,11 +70,7 @@ const ImplementationRow = ({ lines, implementation, index }) => {
         if (
           element.caught === true ||
           element.caught === false ||
-          (element.results &&
-            element.results.every(
-              (each) =>
-                typeof each === "object" && each.hasOwnProperty("errored"),
```

Comment:
I rewrote it to avoid using `typeof` & `hasOwnProperty`. But I was not sure how the data exactly looks like so I might've done it wrong. This should evaluate to `true` if all items in a list have field `errored` equal to `true`, right?

Reply:
Yes, it's totally correct!